deployments/bridges/rialto-millau/entrypoints/relay-headers-rialto-to-millau-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -xeu
+
+sleep 3
+curl -v http://millau-node-alice:9933/health
+curl -v http://rialto-node-alice:9933/health
+
+/home/user/substrate-relay initialize-rialto-headers-bridge-in-millau \
+	--millau-host millau-node-alice \
+	--millau-port 9944 \
+	--rialto-host rialto-node-alice \
+	--rialto-port 9944 \
+	--millau-signer //Alice \
+
+/home/user/substrate-relay rialto-headers-to-millau \
```

Comment:
If this starts to early maybe the previous command should actually wait until the transaction is included?

Reply:
@HCastano I feel this would be more resilient solution than `sleep 6`, maybe let's add log this in the issue tracker, to improve this at some point?

Reply:
That would require changes to the relayer code, so yeah it wouldn't be appropriate to introduce that in this PR. I can open an issue